src/algebra/discrete-log.md

```diff
@@ -132,3 +132,6 @@ int solve (int a, int b, int m) {
 }
 ```
 
+##Problems for Online Judges
+* [Spoj - Power Modulo Inverted](http://www.spoj.com/problems/MOD/)
+* [Topcoder - SplittingFoxes3](https://community.topcoder.com/stat?c=problem_statement&pm=14386&rd=16801)
```

Comment:
I'm not sure the problem which was not solved by tourist during the contest is a good practice material :-) There are probably lots of  more approachable problems on discrete logarithm.